portal/cookbook-guide.md

```diff
@@ -258,6 +258,11 @@ As always, reach out to a Pythia team member for help with any of these steps!
 
 ### Initiate the Cookbook review process
 
+Your Cookbook will be assigned a Cookbook advocate from the Project Pythia maintenance team.
```

Comment:
```suggestion
If you haven't already, now is a great time to [contact the Project Pythia team](https://discourse.pangeo.io/c/education/project-pythia/60) to let them know about your new Cookbook. You will be assigned a Cookbook advocate from the Pythia maintenance team.
```

Reply:
This addresses my concern and I'll approve the PR. Thanks @brian-rose